Client/multiplayer_sa/CMultiplayerSA_FixBadAnimId.cpp

```diff
@@ -111,6 +119,6 @@ void _declspec(naked) HOOK_GetAnimHierarchyFromSkinClump()
 //////////////////////////////////////////////////////////////////////////////////////////
 void CMultiplayerSA::InitHooks_FixBadAnimId(void)
 {
-    EZHookInstall(CAnimBlendAssocGroupCopyAnimation);
+    // EZHookInstall ( CAnimBlendAssocGroupCopyAnimation );
```

Comment:
Is there any reason not to remove this line completely?

Reply:
Nice catch. It should be removed completely.